src/test/scala/uk/gov/hmrc/govukfrontend/views/implicits/RichSelectSpec.scala

```diff
@@ -165,4 +166,50 @@ class RichSelectSpec extends AnyWordSpec with Matchers with MessagesHelpers with
       )
     }
   }
+
+  "Given a Select object, calling asAccessibleAutocomplete" should {
+    "set the accessible autocomplete data attributes" in {
+      val select        = Select()
+      val updatedSelect = select.asAccessibleAutocomplete(Some(AccessibleAutocomplete(Some("Test"))))
+
+      updatedSelect shouldBe Select(attributes =
+        Map(
+          "data-default-value"   -> "Test",
+          "data-show-all-values" -> "false",
+          "data-auto-select"     -> "false",
```

Comment:
worth testing non-default values for the other two attributes here?
```suggestion
      val updatedSelect = select.asAccessibleAutocomplete(Some(AccessibleAutocomplete(Some("Test"), true, true)))

      updatedSelect shouldBe Select(attributes =
        Map(
          "data-default-value"   -> "Test",
          "data-show-all-values" -> "true",
          "data-auto-select"     -> "true",
```